Added initial English localization for documentation (#13)
furiosa-ai/furiosa-sdk: docs/en/source/installation/prerequisites.rst

**********************************
FuriosaAI SDK Installation + Prerequisites
**********************************

.. note::

  The FuriosaAI SDK is composed of a commmand line interface, runtime library, and Python library. 
  The FuriosaAI NPU's kernel driver, firmware, and runtime are distributed according to 
  FuriosaAI's evaluation tool registration and End User License Agreement. 
  For questions about downloading and evaluating the program, please contact us at contact@furiosa.ai.

Minimum Requirements 
=====================================================================
* Ubuntu 18.04 LTS (Bionic Beaver), Debian buster, or higher
* System administrator privileges (root)

> * A Network Environment able to connect to `GitHub <https://github.com/>`_ and `PyPi <https://pypi.org/>`_ 

Reviewer: s/A Network Environment/Internet connection/

.. code-block::

  $ apt-get update
  $ apt-get install cmake build-essential

Installing onnxruntime 1.6.0.
onnxruntime is used for `ONNX <https://onnx.ai/>`_ model format support and model quantization.

.. code-block::

  $ wget https://github.com/hyunsik/onnxruntime/releases/download/v1.6.0/libonnxruntime-1.6.0_amd64.deb
  $ apt-get install -y ./libonnxruntime-1.6.0_amd64.deb

.. _SetupPython:

Configuring the Python SDK execution environment
================================================================

Python 3.7 or above is required to use the FuriosaAI Python SDK.

.. note::

  If you aren't using the FuriosaAI Python SDK, you can skip this section.

.. code-block::

  python --version
  Python 3.8.5

The above command checks the version of Python available on the current system.
If the Python command is not recognized or your system doesn't have a recent enough version of Python, 
you can configure a Python enviroment with one of the following methods.

* :ref:`CondaInstall` (recommended):
  `Conda <https://docs.conda.io/projects/conda/en/latest/index.html>`_ can 
  be used to configure a dedicated Python environment for specific Python applications. 

>   Conda helps to prevent common package dependency issues or Python version issues

Reviewer: ```suggestion
  Conda helps to prevent package dependency conflict issues or Python version issues
```

>   that occur when installing Python applications.

Reviewer: ```suggestion
  that can occur when installing complicated Python applications
```

* :ref:`SetupPythonOnUbuntu`: This option is for users who have a good understanding 
  of the Python execution environment on their Linux system and want to configure 
  the Python environment directly on their system.

.. _CondaInstall:

Configuring the Python environment using Conda
-------------------------------------------------------

You can use Conda to configure a dedicated Python environment for specific Python applications.
Please refer to the documents in `Conda`_ for more information on Conda.

> tart the installation process by downloading the installation program as follows:

Reviewer: tart seems to be a typo error.

Run ``./Miniconda3-latest-Linux-x86_64.sh`` and select `yes`.

.. code-block::

  $ wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
  $ sh ./Miniconda3-latest-Linux-x86_64.sh
  $ source ~/.bashrc
  $ conda --version
  conda 4.8.3

> After installing Anaconda, you can configure an independent Python execution environment 

Reviewer: ```suggestion
After installing Anaconda, you can configure an isolated Python runtime environment 
```
